mentoring/templates/html/mentoring.html

```diff
@@ -1,16 +1,44 @@
-<div class="mentoring">
+<div class="mentoring" data-mode="{{ self.mode }}" data-step="{{ self.step }}">
   <div class="missing-dependency warning" data-missing="{{ self.has_missing_dependency }}">
     You need to complete <a href="{{ missing_dependency_url }}">the previous step</a> before
     attempting this step.
   </div>
-  {% for name, c in named_children %}
-  {{c.body_html|safe}}
-  {% endfor %}
-  {% if self.display_submit %}
-  <input type="button" value="submit" class="submit"></input>
-  <span class="progress" data-completed="{{ self.completed }}">
-    <span class='indicator'></span>
-  </span>
+
+  {% if title %}
+  <div class="title">
+    {% if title %} <h2>{{ title }}</h2> {% endif %}
+  </div>
   {% endif %}
-  <div class="messages"></div>
+
+  <div class="{{self.mode}}-question-block">
+    {{child_content|safe}}
+
+    {% if self.display_submit %}
```

Comment:
I've had a case where display_submit is set to True, but the display button didn't show up:

![screenshot-sandbox opencraft com 2015-02-24 14-32-11](https://cloud.githubusercontent.com/assets/514483/6350194/f670da76-bc31-11e4-9d13-4b072adcfb4b.png)


http://sandbox.opencraft.com/courses/course-v1:TestX+TST-MENTORING-2+now/courseware/f38b4f3ba1c140aeb14daaba27401211/4e69426ce8a149bebcc1fe83b64a6475/2